tools/COMPILE-PROTOS.sh

```diff
@@ -82,7 +82,15 @@ END
 
 # patch the generated GAPIC to send Authorization tokens with insecure requests
 # (this allows the registry command line tools to test container builds)
-sed -i.bak 's/return metadata.NewOutgoingContext(ctx, out)/insecure := os.Getenv("APG_REGISTRY_INSECURE")\ntoken := os.Getenv("APG_REGISTRY_TOKEN")\nif insecure == "1" \&\& token != "" \{ \nout["authorization"] = append(out["authorization"], "Bearer "+token) \n\}\nreturn metadata.NewOutgoingContext(ctx, out)/' gapic/doc.go
+PATCH='\
+insecure := os.Getenv("APG_REGISTRY_INSECURE")\
+token := os.Getenv("APG_REGISTRY_TOKEN")\
+if insecure == "1" \&\& token != "" {\
+  out["authorization"] = append(out["authorization"], "Bearer "+token)\
+}\
+return metadata.NewOutgoingContext(ctx, out)\
+' # no ' or / in the patch; escape & and newlines
```

Comment:
Just curious: what was breaking here? The prior code works for me on Big Sur (11.3).

Reply:
`\n` is just escaped-n, `n`, not a newline, in 10.15